src/components/views/settings/ChangeAvatar.js

```diff
@@ -85,16 +82,15 @@ module.exports = React.createClass({
             newUrl = url;
             if (this.props.onAvatar) {
                 this.props.onAvatar(url);
-            }
-            if (this.props.setAvatar) {
+            } else {
                 if (self.props.room) {
                     return MatrixClientPeg.get().sendStateEvent(
                         self.props.room.roomId,
                         'm.room.avatar',
                         {url: url},
                         ''
                     );
-                } else {
+                } else if (!this.props.groupId) {
```

Comment:
This change is backed out too now in favour of just not using ChangeAvatar at all as it just wasn't adding anything